dali/operators/image/paste/multipaste.cc

```diff
@@ -19,44 +19,86 @@
-.AddOptionalArg<int>("in_anchors", R"code(Absolute coordinates of LU corner
+If the input shapes are uniform and no explicit `output_size` is provided,
+the operator assumes the same output shape (the output canvas size).
+Otherwise, the `output_size` must be specified.
+
+This operator can also change the type of data.)code")
+    .NumInput(1, 64)
+    .AllowSequences()
+    .InputLayout({"HWC", "FHWC"})
+    .AddOptionalArg<std::vector<int>>("in_ids", R"code(Indices of the inputs to paste data from.
+
+If specified, the operator accepts exactly one batch as an input.)code",
+                                      {}, true)
+    .AddOptionalArg<int>("in_anchors", R"code(Absolute coordinates of LU corner
 of the source region.
 
 The anchors are represented as 2D tensors where the first dimension corresponds to the
-elements of ``in_ids`` and the second one is equal to the number of dimensions of the
+number of pasted regions and the second one is equal to the number of dimensions of the
 data, excluding channels.
 
-If not provided, all anchors are zero.)code", nullptr, true)
-.AddOptionalArg<int>("shapes", R"code(Shape of the paste regions.
+If neither `in_anchors` nor `in_anchors_rel` are provided, all anchors are zero.)code",
+                         nullptr, true, true)
+    .AddOptionalArg<float>("in_anchors_rel", R"code(Relative coordinates of LU corner
+of the source region.
+
+The argument works like `in_anchors`, but the values should be floats in `[0, 1]` range,
+describing the anchor placement relative to the input sample shape.)code",
+                           nullptr, true, true)
+    .AddOptionalArg<int>("shapes", R"code(Shape of the paste regions.
 
 The shapes are represented as 2D tensors where the first dimension corresponds to the
-elements of ``in_ids`` and the second one is equal to the number of dimensions of the
+number of pasted regions and the second one is equal to the number of dimensions of the
 data, excluding channels.
 
-If not provided, the shape is calculated so that the region goes from the region anchor
- in the input image until the end of the input image.)code", nullptr, true)
-.AddOptionalArg<int>("out_anchors", R"code(Absolute coordinates of LU corner
-of the destination region.
+If neither `shapes` nor `shapes_rel` are provided, the shape is calculated so that
+the region goes from the region anchor in the input image until the end
+of the input image.)code",
+                         nullptr, true, true)
+    .AddOptionalArg<float>("shapes_rel", R"code(Relative shape of the paste regions.
+
+Works like `shape` argument, but the values should be floats in `[0, 1]` range,
+describing the paste region shape relative to the input shape.)code",
+                           nullptr, true, true)
+    .AddOptionalArg<int>("out_anchors", R"code(Absolute coordinates of LU corner
+of the pasted region in the output canvas.
 
 The anchors are represented as 2D tensors where the first dimension corresponds to the
```

Comment:
```suggestion
The anchors are represented as 2D tensors where the first dimension is equal to the
```
Before it was: _corresponds to the elements of (...)_, but now it doesn`t _correspond_ to the number of pasted regions, but it's equal to it. Right?

Reply:
Equal is less ambigious, I'll adjust it. But functionally, it's the same before and now (just with the multi-input you can't have variadic number of regions per output).